# Name of the joints of the robot
string[] joint_names

# Intensity of the torque that must be applied to each defined joint to "squeeze"
float64[] torque_intensity
